packages/components/README.md

```diff
@@ -5,50 +5,226 @@
 [![Build Status](https://github.com/plone/components/actions/workflows/unit.yml/badge.svg)](https://github.com/plone/components/actions)
 [![Netlify Status](https://api.netlify.com/api/v1/badges/ff1f19ce-9b19-48f9-94a8-d533b53d4a9a/deploy-status)](https://app.netlify.com/sites/plone-components/deploys)
 
-This package contains ReactJS components for use Plone as a headless CMS.
+This package contains ReactJS components for using Plone as a headless CMS.
 
-The purpose of this package is to provide a third party design system free and agnostic set of baseline components to build upon.
+The purpose of this package is to provide an agnostic set of baseline components to build sites upon.
+
+## Design decisions and assumptions
+
+This package provides a set of simple, ready to use components with the following features:
+
+- Agnostic (not tied to any known design system)
+- Composable (they are ready to be the building blocks of other complex components)
+- Theme-able (they can be themed, using both the provided basic theme as a baseline and custom CSS properties)
+- Data-driven-less (they are dumb, presentational components)
+- i18n-less (they do not provide i18n machinery or attached to any i18n framework or library)
+- Built on a renowned headless CMS components library: React Aria Components (RAC) (https://react-spectrum.adobe.com/react-aria/components.html)
+
+Since we are using RAC as the base, the styling of this package components are done via the RAC styling options (https://react-spectrum.adobe.com/react-aria/styling.html).
+See the `react-aria-components` section below for more information about RAC.
+This package provides a simple and basic set of CSS rules to build upon.
+Alternatively, you can bring your own styles, removing the basic styling or complementing it, and build on top of it.
+You can even use RAC to use other CSS utility libraries, like TailwindCSS.
+
+## Styling
+
+This package provide a basic set of CSS rules.
+We call them "BasiQ" (the "Q" is for "Quanta").
```

Comment:
@sneridagh @stevepiercy  No one asked my opinion but here I am giving it anyhow :) 
I am not so sure if it would be a good thing to have a name that is like an acronym.
I am thinking of how we are going to spell it when we give training about it.
Personally I wouldn't try to make clever names in trying to combine something
and giving it meaning then somehow.

I also think we should investigate if the naming doesn't clash with other  entities that use
this naming, performing a google search gave me a few results for basiq such as 
https://www.basiq.io/
I don't know how far are trademarks and copyright nowadays but it doesn't hurt for us todo
some searching and ensuring we don't use something already established and copyrighted

Reply:
I had a similar moment of "hmmm", how do you even pronounce that? "Bay-sik" or "Bay-suh-que"?

"Basiq" is trademarked.

Let's avoid the attorney horde. I'll push a commit to replace it with "Quanta base CSS rules" and grammatically correct phrases.

Reply:
On second thought, I won't push a commit. It means renaming the package.

Reply:
@stevepiercy 
I've already talked about this with @ichim-david 
I came up with the name in 5mins. It's an internal name for now, we won't use it anywhere else. 
We will sell `@plone/components` what's inside stays in Vegas.
Let's maintain it as internal name until someone has a better name. And yes, we need a name for them.
Quanta is also something else out there, nobody complained.

Reply:
@stevepiercy 
OTOH, I after giving it a second thought, change a Q for a c is easy anyways, but let's all think in the name of the basic/fundamental/foundational CSS rules.

Reply:
@sneridagh indeed, I think clever spelling in names end up being a footgun. I think it is better not to brand something that you do not intend to put much marketing effort toward. It's just confusing to developers.

Reply:
Ready.